src/analysis/retail/shaman/elemental/constants.tsx

```diff
@@ -21,3 +21,7 @@ export const ON_CAST_BUFF_REMOVAL_GRACE_MS = 50;
  * in the guide section.  For use with `ExplanationAndDataSubSection`.
  */
 export const GUIDE_EXPLANATION_PERCENT_WIDTH = 40;
+
+export const ELECTRIFIED_SHOCKS_DURATION = 9000;
+export const ELEMENTAL_BLAST_COST = 75;
```

Comment:
Feedback from HawkCorrigan:

This should prolly be done correctly with EotS (talent that reduces the cost of these spells).

Reply:
Agree.

Did some codebase searching and found this analyzer called "SpellResourceCost". Using this is a much better solution than what I have here.